ably.gemspec

```diff
@@ -20,40 +20,28 @@ Gem::Specification.new do |spec|
 
   spec.add_runtime_dependency 'eventmachine', '~> 1.2.6'
   spec.add_runtime_dependency 'em-http-request', '~> 1.1'
-  spec.add_runtime_dependency 'statesman', '~> 8.0'
+  spec.add_runtime_dependency 'statesman', '~> 9.0'
   spec.add_runtime_dependency 'faraday', '~> 1.0'
   spec.add_runtime_dependency 'typhoeus', '~> 1.4'
-
-  if RUBY_VERSION.match(/^1\./)
-    spec.add_runtime_dependency 'json', '< 2.0'
-  else
-    spec.add_runtime_dependency 'json'
-  end
+  spec.add_runtime_dependency 'json'
   spec.add_runtime_dependency 'websocket-driver', '~> 0.7'
   spec.add_runtime_dependency 'msgpack', '>= 1.3.0'
   spec.add_runtime_dependency 'addressable', '>= 2.0.0'
 
   spec.add_development_dependency 'rake', '~> 13.0'
   spec.add_development_dependency 'redcarpet', '~> 3.3'
-  spec.add_development_dependency 'rspec', '~> 3.10.0'
+  spec.add_development_dependency 'rspec', '~> 3.11.0'
   spec.add_development_dependency 'rspec-retry', '~> 0.6'
   spec.add_development_dependency 'yard', '~> 0.9'
   spec.add_development_dependency 'rspec-instafail', '~> 1.0'
   spec.add_development_dependency 'bundler', '>= 1.3.0'
-
-  if RUBY_VERSION.match(/^1\./)
-    spec.add_development_dependency 'public_suffix', '~> 1.4.6' # Later versions do not support Ruby 1.9
-    spec.add_development_dependency 'webmock', '2.2'
-    spec.add_development_dependency 'parallel_tests', '~> 2.9.0'
-  else
-    spec.add_development_dependency 'webmock', '~> 3.11'
-    spec.add_development_dependency 'simplecov', '~> 0.21.2'
-    spec.add_development_dependency 'simplecov-lcov', '~> 0.8.0'
-    spec.add_development_dependency 'parallel_tests', '~> 3.7'
-    if !RUBY_VERSION.match(/^2\.[0123]/)
-      spec.add_development_dependency 'pry', '~> 0.14.1'
-      spec.add_development_dependency 'pry-byebug', '~> 3.8.0'
-    end
+  spec.add_development_dependency 'webmock', '~> 3.11'
+  spec.add_development_dependency 'simplecov', '~> 0.21.2'
+  spec.add_development_dependency 'simplecov-lcov', '~> 0.8.0'
+  spec.add_development_dependency 'parallel_tests', '~> 3.8'
+  if !RUBY_VERSION.match(/^2\.[0123]/)
```

Comment:
I think we can safely remove this line since CI doesn't use this versions

Reply:
@lukaszsliwa I mean it was added because there was some problem with pry on version 2.3 and below. Now we can just add it unconditionally

Reply:
Okey @TheSmartnik I will update this later. Generally this PR is in draft mode. Not ready for review yet as faraday 2 does not work properly